metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -380,7 +380,8 @@ class TableauConfig(
 
     fetch_size: int = Field(
         default=250,
-        description="Specifies the number of records to retrieve in each batch during a query execution.",
+        description="[deprecated] Use page_size instead. Specifies the number of records to retrieve in each batch during a query execution.",
```

Comment:
with this change, fetch_size does not do anything, right?

so we should use `pydantic_field_removed` and make it clear that only page size has an impact

Reply:
@hsheth2 

I haven't found `pydantic_field_removed` in pydantic documentation. Could you please add additional detail here for the same.

Reply:
https://github.com/datahub-project/datahub/blob/8cf778dc9b2bb77f9af1ffd979e65fb283579938/metadata-ingestion/src/datahub/configuration/validate_field_removal.py#L10

Reply:
Oh!. Yesterday I searched the code base as per your old comment `pydantic_field_removed `. Got it now

Reply:
done